src/components/TradeWidget/utils.ts

```diff
@@ -7,19 +7,21 @@ import { encodeTokenSymbol } from '@gnosis.pm/dex-js'
 import { BATCH_START_THRESHOLD } from './validationSchema'
 import { BATCH_TIME_IN_MS } from 'const'
 import { TokenDetails } from 'types'
-import { formatPriceToPrecision } from 'components/trade/PriceSuggestions/PriceSuggestionItem'
 
-export function calculateReceiveAmount(priceValue: string, sellValue: string): string {
+export function calculateReceiveAmount(priceValue: string, sellValue: string, receiveTokenPrecision: number): string {
   let receiveAmount = ''
   if (priceValue && sellValue) {
     const sellAmount = parseBigNumber(sellValue)
     const price = parseBigNumber(priceValue)
 
     if (sellAmount && price) {
-      const receiveBigNumber = sellAmount.dividedBy(price)
+      const receiveBigNumber = sellAmount.times(price)
+      const receiveAmountFormatted = receiveBigNumber.decimalPlaces(receiveTokenPrecision, 1)
```

Comment:
why is this called formatted if it's a bignumber?

Reply:
good point, renamed